themes/digital.gov/src/scss/new/_join-community-form.scss

```diff
@@ -0,0 +1,118 @@
+@use "uswds-core" as *;
+
+.join-community-form{
+	@include u-margin-bottom(5);
+    @include u-padding-top(2.5);
+    @include u-padding-bottom(4);
+	@include u-padding-x(3);
+	@include u-text('light');
+	@include u-radius('sm');
+	h4{
+		@include u-margin-bottom('105');
+        @include u-margin-top(0);
+		@include u-font('sans', 'lg');
+		@include u-text('normal');
+		i{
+			@include u-margin-right('05');
+		}
+        svg{
+			@include u-margin-right(2.5);
+        }
+	}
+	a{
+		@include u-text('no-underline');
+		&:hover{
+			@include u-text('no-underline');
+		}
+	}
+
+	.highlight{
+		@include u-bg('yellow-5');
+	}
+
+    .type {
+        @include u-padding-bottom(2);
+    }
+
+	.join-box{
+		@include u-margin-left(0);
+	}
+
+    .join-box-wrapper{
+        p{
+            @include u-padding-bottom(2);
+        }
+    }
+
+	strong{
+		@include u-text('medium');
+	}
+	p:last-child{
+		@include u-margin-bottom(0);
+	}
+	blockquote{
+		@include u-padding-left(0);
+		@include u-border(0);
+	}
+	ol,ul{
+		@include u-margin-y(0);
+		@include u-padding-left('205');
+		li{
+			@include u-margin-bottom(2);
+		}
+	}
+}
+
+
+// For notes within list items
+li .join-community-form{
+	@include u-padding-y('05');
+	@include u-padding-x(1);
+}
+
+
+// Note — color settings
+.join-community-form{
+	@include u-bg('blue-warm-50v');
+	@include u-text('gray-1');
+	@include u-border-left('05', 'solid', 'blue-20');
+	a{
+		@include u-text('blue-70');
+		&:hover{
+			@include u-text('blue-80');
+		}
+	}
+	.highlight{
+		@include u-bg('blue-10');
+	}
+
+    button {
+        @include u-bg('gray-1');
+        &:hover{
+            @include u-bg('blue-5');
+        }
+        &:active{
+            @include u-bg('blue-10');
+        }
+    }
+
+    .join-button {
+        @include u-padding-top(1);
+    }
+
+    .join{
+        @include u-margin-bottom(2);
+        @include u-padding-y('105');
+        @include u-padding-x(3);
+        @include u-text('left');
+        @include u-text('blue-warm-50v');
+        @include u-text('no-underline');
+        @include u-font('sans', 'md');
+        @include u-bg('gray-1');
+        @include u-radius('md');
+        @include u-text('bold');
+        &:hover{
+          @include u-bg('blue-5');
+        }
+    }
+}
```

Comment:
There's a mix of spaces and tabs used, let's use the standard 2 spaces for tabs and fix the formatting.

My editor shows tabs are being used. Notice the mix on dots (spaces) and arrows (tabs) in screenshot below:
![image](https://user-images.githubusercontent.com/3385219/211033500-db1cb47c-2bf7-4dac-9c7d-3856ad84cf9a.png)


You can see by enabling rendering on all whitespace characters in VS Code settings:
![image](https://user-images.githubusercontent.com/3385219/211033443-a6cdb92b-5463-4944-ac16-a443402a8490.png)

Reply:
Added prettier to VsCode formatting to ensure this doesn't happen, thanks!